src/lib/callApi.js

```diff
@@ -4,27 +4,38 @@ import { getLevelForContributionCount } from './Levels';
 const config = require('./config');
 
 let localData = [];
+/* eslint-disable global-require */
 try {
-  // eslint-disable-next-line global-require
   localData = require('./msf-mapswipe-users-export.json');
   // eslint-disable-next-line no-console
 } catch (e) { console.error(e); }
 
 // If eq 'dev' fetch data locally else (eq 'prod') fetch data from Firebase
-const SOURCE = 'dev';
+const SOURCE = 'prod';
 
 firebase.initializeApp(config);
 const db = firebase.database();
 
-const isInclude = (str, pattern) => str.toLowerCase().includes(pattern.toLowerCase());
+const isInclude = (str, pattern, isSearcAtStart) => isSearcAtStart
+  ? str.toLowerCase().startsWith(pattern.toLowerCase())
+  : str.toLowerCase().includes(pattern.toLowerCase());
 
-export const getLocalData = (query = undefined) => {
+const getCompanyLogo = (username) => {
+  switch (username) {
+    case (username.match(/^sf_/i) || {}).input:
+      return require('../assets/companies/salesForce.png');
```

Comment:
Move these `requires` to the top of the file and assign them variables - that way it's more clear what this file depends on, as well as not having to re-require them every time this function gets run. Unless there's a necessary reason for this that I'm missing?